test/commands/pulls/close.test.js

```diff
@@ -0,0 +1,78 @@
+const yargs = require("yargs")
+const nock = require('nock');
+const yargsModule = require("../../../src/commands/pulls/close")
+
+// Don't let Octokit make network requests
+nock.disableNetConnect();
+
+jest.mock('fs', () => ({
+	existsSync: jest.fn().mockReturnValue(true),
+	readFileSync: jest.fn().mockReturnValue(true),
+}))
+jest.spyOn(global.console, "warn");
+afterEach(() => {
+	jest.clearAllMocks();
+});
+
+describe("Yargs", () => {
+	test("`pulls close` command module exports an object that can be used by yargs", () => {
+		expect(yargsModule).toEqual(
+			expect.objectContaining({
+				command: expect.stringMatching("close"),
+				desc: expect.any(String),
+				builder: expect.any(Function),
+				handler: expect.any(Function),
+			})
+		)
+	})
+
+	test("yargs can load the `pulls close` command without any errors or warnings", () => {
+		expect(() => {
+			yargs.command(
+				yargsModule.command,
+				yargsModule.desc,
+				yargsModule.builder,
+				yargsModule.handler
+			).argv
+		}).not.toThrow()
+		expect(console.warn).not.toBeCalled()
+	})
+
+	const requiredOptions = {
+		token: "test",
+		owner: "test",
+		repo: "test",
+		number: 1,
+	}
+	for (let option of Object.keys(requiredOptions)) {
+		test(`Running the command handler without '${option}' throws an error`, async () => {
+			expect.assertions(1)
+			try {
+				const testOptions = Object.assign({}, requiredOptions)
+				delete testOptions[option]
+				await yargsModule.handler(testOptions)
+			} catch (error) {
+				expect(error).toBeInstanceOf(Error)
+			}
+		})
+	}
```

Comment:
As mentioned in my other comment, I think you could use `yargs.parse` here to test the validation of options.